Render markdown response

ankushdutt opened the pull request:
This PR adds support for rendering Markdown in chat messages by utilizing [react-markdown](https://github.com/remarkjs/react-markdown).

Also, adds [@tailwindcss/typography](https://tailwindcss.com/docs/typography-plugin) to add beautiful default styles to rendered HTML elements from Markdown. The default styles can be found here: https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/styles.js

**Before**
![Screenshot 2023-07-31 at 1 32 38 AM](https://github.com/cofactoryai/textbase/assets/51174480/9f07d346-77e6-479e-a493-c49cd4c945fb)

**After**
![Screenshot 2023-07-31 at 1 32 58 AM](https://github.com/cofactoryai/textbase/assets/51174480/d2aa4126-1fe3-4282-b484-66c529a3159f)

kaus-cofactory commented:
Hey @ankushdutt, this is a really thoughtful change. However I'd appreciate if you can also upload the new `npm build` of this frontend changes. It should be in a `dist` folder inside frontend directory.

Can you do this and see if it all works. Also, I think the `package-lock.json` is not really required so can you delete/omit it while pushing (its just increasing the number of lines of code change 👀)

ankushdutt commented:
Hi @kaus-cofactory, thanks for the review. I have now updated my commit.

Indeed the `package-lock.json` is not required since it will be automatically updated using `npm install`, and I have removed it in the last commit. However, it was intentional to make sure the that the package versions are same for everyone. See: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

kaus-cofactory commented:
LGTM ✅, yes we can include the lockfile but since in our case the frontend will not remain fixed and will change over time as we keep adding and improving features to the library.
